assets/report/input.css

```
@@ -264,6 +264,23 @@
+  .btn-ghost:disabled {
+    @apply cursor-not-allowed opacity-40;
+  }
+  .btn-ghost:disabled:hover {
+    @apply bg-card text-muted-foreground;
+  }
+  .nav-history {
+    @apply inline-flex items-center gap-1;
+  }
+  .nav-history-btn svg {
+    @apply h-4 w-4;
+    fill: none;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)